tests/robustness/multiclient_test/multiclient_test.go

```diff
@@ -52,10 +52,16 @@ func TestManySmallFiles(t *testing.T) {
 		require.NoError(t, err)
 
 		_, err = eng.ExecAction(ctx, engine.DeleteRandomSubdirectoryActionKey, deleteDirOpts)
-		require.NoError(t, err)
+		// ignore the dir-not-found errors
```

Comment:
Create a helper for executing and ignoring the no-op error for delete-type actions, and then call that helper here and below.

Reply:
done